pom.xml

```diff
@@ -0,0 +1,257 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.sonatype.oss</groupId>
```

Comment:
@ddaaggeett Hah I didn't even know it was deprecated! So nope I haven't worried about it. It seems that the alternative now is just to pull some of the settings from their pom.xml directly into your own pom.xml. I think its probably not worth worrying about just yet, can always improve that later...